### PR TITLE
LPS-193823 - Add Client Secret and Client Id as fields to CTRemote and ctRemoteId to CTCollection

### PR DESCRIPTION
**Pull request by david-truong**

### CODE REVIEW

**Comment by david-truong** [comment]
ci:test:sf

**Comment by david-truong** [comment]
ci:stop

**Comment by david-truong** [comment]
ci:test:relevant

**Comment by david-truong** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:test:relevant

**Comment by david-truong** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:test:relevant

**Comment by david-truong** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:test:relevant

**Comment by david-truong** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:test:relevant

**Comment by david-truong** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:stop

File: portal-web/test/functional/com/liferay/portalweb/macros/json/Publications/JSONPublicationsAPI.macro
Comment (david-truong) [inline portal-web/test/functional/com/liferay/portalweb/macros/json/Publications/JSONPublicationsAPI.macro:1]: @suilin this was the problem.  is this fix okay?

**Comment by david-truong** [comment]
ci:test:relevant

**Comment by david-truong** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:stop

File: portal-web/test/functional/com/liferay/portalweb/macros/json/Publications/JSONPublicationsAPI.macro
Comment (suilin) [reply portal-web/test/functional/com/liferay/portalweb/macros/json/Publications/JSONPublicationsAPI.macro:1]: @david-truong, I checked it locally, your last commit has fixed the issue, should we retrigger the publish suite?

Comment (david-truong) [reply portal-web/test/functional/com/liferay/portalweb/macros/json/Publications/JSONPublicationsAPI.macro:1]: I restarted it here: https://github.com/liferay-publications/liferay-portal/pull/676

**Pull request closed without merging**